### DISCUSSION TITLE
Task view issue

### DISCUSSION THREAD

**Discussion by Adhy86** [valinet/ExplorerPatcher#1857] [General]
Taskbar style : Win 10
After I'am update my windows, when I click "task view" on taskbar no respon.
Please fix this.

Thanks

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1687

Please close this one.

**Comment by Poopooracoocoo** [comment]
Closing as duplicate of https://github.com/valinet/ExplorerPatcher/issues/1687 and https://github.com/valinet/ExplorerPatcher/issues/1802
